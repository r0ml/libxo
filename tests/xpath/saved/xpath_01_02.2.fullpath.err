main: input '#run --libxo @fullpath:slax,pretty'
main: input '#run'
main: input '?one[two == 4]'
--- 0: 10
    000010: type 74 (C_PATH), str 0 [], contents 1, next 0, prev 0 
        000001: type 70 (C_ELEMENT), str 1 [one], contents 2, next 0, prev 0 
            000002: type 75 (C_PREDICATE), str 0 [], contents 4, next 0, prev 0 
                000004: type 14 (L_EQUALS), str 11 [==], contents 5, next 0, prev 0 
                    000005: type 74 (C_PATH), str 0 [], contents 3, next 6, prev 0 
                        000003: type 70 (C_ELEMENT), str 7 [two], contents 0, next 0, prev 0 
                    000006: type 53 (T_NUMBER), str 14 [4], contents 0, next 0, prev 5 
main: status: zero
main: input '+one'
main: status: track
main: input '$two 4'
main: key: 'two'='4'
main: status: full
main: input '=output yes'
main: field: 'output'='yes'
main: status: full
main: input '-one'
main: status: track
main: input 'r'
main: status: zero
main: input '?one[two == 40]'
--- 0: 10
    000010: type 74 (C_PATH), str 0 [], contents 1, next 0, prev 0 
        000001: type 70 (C_ELEMENT), str 1 [one], contents 2, next 0, prev 0 
            000002: type 75 (C_PREDICATE), str 0 [], contents 4, next 0, prev 0 
                000004: type 14 (L_EQUALS), str 11 [==], contents 5, next 0, prev 0 
                    000005: type 74 (C_PATH), str 0 [], contents 3, next 6, prev 0 
                        000003: type 70 (C_ELEMENT), str 7 [two], contents 0, next 0, prev 0 
                    000006: type 53 (T_NUMBER), str 14 [40], contents 0, next 0, prev 5 
main: status: zero
main: input '+one'
main: status: track
main: input '$two 4'
main: key: 'two'='4'
main: status: track
main: input '=output nope'
main: field: 'output'='nope'
main: status: track
main: input '-one'
main: status: track
main: input 'r'
main: status: zero
main: input '?one/two[three == 5]'
--- 0: 12
    000012: type 74 (C_PATH), str 0 [], contents 1, next 0, prev 0 
        000001: type 70 (C_ELEMENT), str 1 [one], contents 0, next 3, prev 0 
        000003: type 70 (C_ELEMENT), str 7 [two], contents 4, next 0, prev 1 
            000004: type 75 (C_PREDICATE), str 0 [], contents 6, next 0, prev 0 
                000006: type 14 (L_EQUALS), str 19 [==], contents 7, next 0, prev 0 
                    000007: type 74 (C_PATH), str 0 [], contents 5, next 8, prev 0 
                        000005: type 70 (C_ELEMENT), str 13 [three], contents 0, next 0, prev 0 
                    000008: type 53 (T_NUMBER), str 22 [5], contents 0, next 0, prev 7 
main: status: zero
main: input '+one'
main: status: track
main: input '+two'
main: status: track
main: input '$three 5'
main: key: 'three'='5'
main: status: full
main: input '=output sure'
main: field: 'output'='sure'
main: status: full
main: input '-two'
main: status: track
main: input '=output no'
main: field: 'output'='no'
main: status: track
main: input '-one'
main: status: track
main: input 'r'
main: status: zero
main: input '?one[two == 4]/three[four == 5]'
--- 0: 19
    000019: type 74 (C_PATH), str 0 [], contents 1, next 0, prev 0 
        000001: type 70 (C_ELEMENT), str 1 [one], contents 2, next 10, prev 0 
            000002: type 75 (C_PREDICATE), str 0 [], contents 4, next 0, prev 0 
                000004: type 14 (L_EQUALS), str 11 [==], contents 5, next 0, prev 0 
                    000005: type 74 (C_PATH), str 0 [], contents 3, next 6, prev 0 
                        000003: type 70 (C_ELEMENT), str 7 [two], contents 0, next 0, prev 0 
                    000006: type 53 (T_NUMBER), str 14 [4], contents 0, next 0, prev 5 
        000010: type 70 (C_ELEMENT), str 20 [three], contents 11, next 0, prev 1 
            000011: type 75 (C_PREDICATE), str 0 [], contents 13, next 0, prev 0 
                000013: type 14 (L_EQUALS), str 33 [==], contents 14, next 0, prev 0 
                    000014: type 74 (C_PATH), str 0 [], contents 12, next 15, prev 0 
                        000012: type 70 (C_ELEMENT), str 28 [four], contents 0, next 0, prev 0 
                    000015: type 53 (T_NUMBER), str 36 [5], contents 0, next 0, prev 14 
main: status: zero
main: input '+one'
main: status: track
main: input '$two 4'
main: key: 'two'='4'
main: status: track
main: input '+three'
main: status: track
main: input '$four 5'
main: key: 'four'='5'
main: status: full
main: input '=output 1'
main: field: 'output'='1'
main: status: full
main: input '-three'
main: status: track
main: input '+three'
main: status: track
main: input '$four 6'
main: key: 'four'='6'
main: status: track
main: input '=output 0'
main: field: 'output'='0'
main: status: track
main: input '-three'
main: status: track
main: input '+three'
main: status: track
main: input '=output should-not'
main: field: 'output'='should-not'
main: status: track
main: input '-three'
main: status: track
main: input '+three'
main: status: track
main: input '$four 5'
main: key: 'four'='5'
main: status: full
main: input '=output finally'
main: field: 'output'='finally'
main: status: full
main: input '-three'
main: status: track
main: input '-one'
main: status: track
