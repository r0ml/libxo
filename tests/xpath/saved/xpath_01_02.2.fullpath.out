/one[two='4']/output='yes'
/one/two[three='5']/output='sure'
/one[two='4']/three[four='5']/output='1'
/one[two='4']/three[four='5']/output='finally'
